packs/feats/plague-rat.json

```
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Your bites carry a cursed infection. When you hit and deal damage to a creature with a jaws Strike while in hybrid or rat shape, the creature is cursed until the start of your next turn. Whenever they regain Hit Points during that time, they must attempt a Fortitude save against your class DC or spell DC, whichever is higher.</p>\n<p><strong>Success</strong> The creature regains Hit Points normally.</p>\n<p><strong>Failure</strong> The creature regains half the normal Hit Points.</p>\n<p><strong>Critical Failure</strong> The creature regains no Hit Points.</p>"
+            "value": "<p>Your bites carry a cursed infection. When you hit and deal damage to a creature with a jaws Strike while in hybrid or rat shape, the creature is cursed until the start of your next turn. Whenever they regain Hit Points during that time, they must attempt a @Check[fortitude|against:class-spell|rollerRole:target|showDC:all] save against your class DC or spell DC, whichever is higher.</p><hr /><p><strong>Success</strong> The creature regains Hit Points normally.</p>\n<p><strong>Failure</strong> The creature regains half the normal Hit Points.</p>\n<p><strong>Critical Failure</strong> The creature regains no Hit Points.</p>"
```

```suggestion
            "value": "<p>Your bites carry a cursed infection. When you hit and deal damage to a creature with a jaws Strike while in hybrid or rat shape, the creature is cursed until the start of your next turn. Whenever they regain Hit Points during that time, they must attempt a @Check[fortitude|against:class-spell] save against your class DC or spell DC, whichever is higher.</p><hr /><p><strong>Success</strong> The creature regains Hit Points normally.</p>\n<p><strong>Failure</strong> The creature regains half the normal Hit Points.</p>\n<p><strong>Critical Failure</strong> The creature regains no Hit Points.</p>"
```
Don't need to specify the roller role since that's the default behavior for a save, and in case a GM builds a werecreature NPC using PC rules we don't want to leak info, so just keeping to the default of owner should be fine.